crates/bevy_ui/src/ui_node.rs

```diff
@@ -1714,3 +1714,12 @@ mod tests {
         assert_eq!(GridPlacement::end_span(-4, 12).get_start(), None);
     }
 }
+
+#[derive(Component, Clone, Debug, Reflect, Eq, PartialEq)]
+pub struct UiCamera(pub Entity);
```

Comment:
A doc comment here would be nice

Reply:
I am also considering a better naming, something like `TargetCamera`. What do you think?

Reply:
That's definitely better, `UiCamera` sounds like a component for the camera entity itself. I can also think of `TargetUiCamera` to be more specific, but it could be confusing since it points to a `Camera2d` or `Camera3d`. If you go with `TargetCamera`, just be explicit in the documentation that it is only meant for UI nodes. I'm afraid I don't have enough of a naming sense to give more suggestions :smile:

Reply:
Done, together with the rest of the tiny fixes

Reply:
As an alternative, it could potentially also be `UiRoot`. My understanding is that the Entity contained within this struct is the one that the `Camera` component is on?

Reply:
`Entity` here indeed points to the one with `Camera` component on it. I think `UiRoot` may be misleading because of it. Also, this component is later automatically propagated to children, so all Nodes will have `UiRoot` on them.